class LogInError(Exception):
    pass


class NotConnectedError(Exception):
    pass